src/adv.py

```diff
@@ -49,3 +65,36 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+valid_cmds = ['s', 'n', 'e', 'w', 'q']
+
+game = True
+
+while game:
+    # print the current room name and description
```

Comment:
Great work with the while loop here. You could have wrote `while True`, but this is even better because now you have control of the loop end without using `break`. Reminds me of setting the state in React, only here its the state of the game!